src/screens/flows/WalletConnect/WaitingBridge/index.js

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from 'react';
+import { ActivityIndicator, Text, View } from 'react-native';
+import { useDispatch } from 'react-redux';
+
+import { Container } from '@/layout';
+import { updateBridgeTimeout } from '@/redux/slices/walletconnect';
+import { useNavigation } from '@/utils/navigation';
+
+import styles from '../styles';
+
+function WalletConnectWaitingBridge() {
```

Comment:
```suggestion
function WalletConnectWaitingBridge({ navigation, route }) {
```
you can use the navigation and route objects from params. Same for other screens